translations/TablerBundle.fr.xlf

```diff
@@ -146,6 +146,18 @@
         <source>or</source>
         <target state="translated">ou</target>
       </trans-unit>
+      <trans-unit id="465zeJ" resname="Show notifications">
+          <source>Show notifications</source>
+          <target>Afficher notifications</target>
+      </trans-unit>
+      <trans-unit id="F5Dfzd" resname="You have no notification">
+          <source>You have no notification</source>
+          <target>Vous avez aucune notification</target>
+      </trans-unit>
+      <trans-unit id="Pf48Zd" resname="And %count% more...">
+          <source>And %count% more...</source>
+          <target>Et %count% autres...</target>
```

Comment:
https://www.htmlsymbol.com/html-entities/horizontal-ellipsis.html

Reply:
Never thought of using it.
Do people from weblate will do it too?

Reply:
I hope so 😁 Translators usually know these things, actually I learned it from one of them.
I just think it looks better than three singled dots